sup3r/preprocessing/feature_handling.py

```diff
@@ -724,6 +725,98 @@ def compute(cls, data, height):
         return rews
 
 
+class UWindPowerLaw(DerivedFeature):
+    """U wind component feature class with needed inputs method and compute
+    method. Uses power law extrapolation to get values above surface
+
+    https://en.wikipedia.org/wiki/Wind_profile_power_law
```

Comment:
Thoughts on using this NOAA reference and set alpha=0.2 and put alpha as a class variable for transparency?

https://csl.noaa.gov/projects/lamar/windshearformula.html

Reply:
Class var is a good idea. Why the disagreement between the alpha values though?

Reply:
I'm not sure but NOAA is more defensible haha

Reply:
our high school teachers would be so proud

Reply:
Our high school teachers were wrong. Wikipedia actually links to a paper. NOAA doesn't have a source for 0.2 that I can see?